chocolatey/choco-theme
# (#335)(#346) Add Multiple Tab Support and Default Badge Color

## Description Of Changes
Sometimes, multiple tabs in different spots of the page need triggered at the same time. An example of this use case would be on docs.chocolatey.org and separating Chocolatey v1 information from Chocolatey v2.

This is achieved by adding the data-choco-tab-multi='{"${cookieName}": "${cookieValue}"}' attribute to each .nav-link with the same corresponding value of those that need to be triggered.

This updates the default badge color to change to the accessible text
color for each theme. For example, in light mode the text will be black,
and in light mode the text will be white. 

## Motivation and Context
This will help users navigate to the most relevant information faster. 

## Testing
1. [Run this PR in the choco-design-system](https://github.com/chocolatey/choco-design-system/pull/24)
2. Navigate to http://localhost:5080/components/tabs#trigger-multiple-tabs-at-once and play with the tabs in that example. Reload the page to see the preference saving.
3. Navigate to http://localhost:5080/components/badges and notice the last badge says "Text Theme" and will have no background and will be black color in light mode and white in dark mode. 

### Operating Systems Testing
n/a

## Change Types Made

## Change Checklist

## Related Issue
* ENGTASKS-3197
* https://github.com/chocolatey/docs/issues/752
* https://github.com/chocolatey/choco-design-system/issues/23
* https://github.com/chocolatey/choco-design-system/issues/25
* #335 
* #346
